.circleci/config.yml

```diff
@@ -5,6 +5,54 @@ orbs:
   flutter: circleci/flutter@2
   browser-tools: circleci/browser-tools@1
   codecov: codecov/codecov@3
+  node: circleci/node@5.1.0
+  flutter-install:
+    orbs:
+      android: circleci/flutter@2
+    commands:
+      install_sdk:
+        description: Custom script to fix issue in Flutter orb
```

Comment:
Which issue is this fixing?

Reply:
This is mentioned in the PR description - There was also another [issue](https://discuss.circleci.com/t/failed-to-run-python-script-directly-pyenv-version-name-command-not-found/37526/2) with the CI where the [Flutter Orb](https://github.com/CircleCI-Public/flutter-orb/blob/main/src/scripts/install-sdk.sh#L21) which was exporting the flutter binary in a way that affected. To fix this we have copied the bash script and patched the required changes

Reply:
The linked forum thread is about a Python script. Could you please expand about the issue in the Flutter Orb?

Reply:
You can see in the next link that the flutter-orb is also making the same mistake as mentioned. The flutter-orb is adding the flutter binary to the path in a way which affects other paths. This link mentioned in the forum explains it better - https://github.com/CircleCI-Public/gcp-cli-orb/issues/22#issuecomment-715463423

Reply:
Thanks for the context. Wasn't that fixed here https://github.com/CircleCI-Public/flutter-orb/pull/28/files#diff-8f89d2af8215d6241d3412ffb02561f587288984cf9de7cacb93e5df1177e81eR21?

Reply:
(They escaped the `$` in `$PATH`)

Reply:
Great catch, looks like I was using older version of the orb for testing. This is fixed now and the description is updated